databases/week1/database-week1.sql

```diff
@@ -0,0 +1,58 @@
+DROP DATABASE IF EXISTS hyf_db;
+
+CREATE DATABASE IF NOT EXISTS hyf_db;
+
+USE hyf_db;
+
+-- 1.Find out how many tasks are in the task table
+SELECT * 
```

Comment:
To get number of tasks use Count(*)

Reply:
Hej @ReenuMary . thanks for comment on my homework , after pushing i  realized  that. i will do as comment thank you ....:)